src/backend/redis/backend.cr

```diff
@@ -56,7 +56,6 @@ module Cable
       return if redis_subscribe.nil?
 
       redis_subscribe.subscribe(stream_identifier)
-      redis_subscribe.flush
```

Comment:
It looks like this `subscribe` method is already calling `flush` 

https://github.com/jgaskins/redis/blob/8d177adb4197c11fab7a07f8bf8dce2a98988f01/src/connection.cr#L185-L188

@jgaskins  would you be able to confirm this?

Reply:
Confirmed 👍🏼 It's what lets you change up the subscriptions in the middle of your subscription block without manually flushing the connection:

```crystal
redis.subscribe "asdf" do |subscription, connection|
  subscription.on_message do |channel, msg|
    if new_channel = should_subscribe?(channel, msg)
      connection.subscribe new_channel
    end

    # ...
  end
end
```